src/site/content/en/blog/performance_tool_score_diffs.md

```diff
@@ -0,0 +1,179 @@
+  </table>
+  <p>*other metrics might be measured by the tools, but they aren’t included in performance/speed score calculations<br>**TMS uses FID as a tiebreaker in cases where FCP between compared sites is the same (but FCP is the lead metric)</p>
+</div>
+
+
+# Tool Sections: Breakdown
+
+## Test My Site (TMS)
+TMS is a ‘Think with Google’ tool that allows you to compare your FCP to that of your competitors and estimate the monetary value of improving your site’s performance.
+
+### Mobile Site Speed: Field Data, FCP Only, Site Level
+The first part of the report, the mobile site speed seconds figure === First Contentful Paint (FCP) of the origin from CrUX.
+
+Site speed is the speed of <b>all the pages</b> that make up your site.
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="F1_tms_sitespeed.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 1 — Mobile Site Speed (Field Data, FCP Only, Site Level)
+  </figcaption>
+</figure>
+
+### Site Speed Tools: Field Data, FCP Only, Site Level
+In the second part of the report, the mobile site speed seconds figure === First Contentful Paint (FCP) of the origin from CrUX. 
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="F2_tms_sitespeed.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 1 — Site Speed Tools (Field Data, FCP Only, Site Level)
+  </figcaption>
+</figure>
+
+### Page Speed Tools: Lab Data, FCP Only, URL Level
+The third part of the report, the page speed seconds figure === First Contentful Paint (FCP) of the page (not the site) from Lighthouse. 
+
+The page speed tool calculates the speed of <b>individual pages</b> on your site.
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="F3_tms_pagespeed.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 1 — Page Speed Tools (Lab Data, FCP Only, URL Level)
+  </figcaption>
+</figure>
+
+FAQs for TMS can be found <a href="https://www.thinkwithgoogle.com/feature/testmysite/faq">here</a>.
+
+<hr>
+
+## PageSpeed Insights (PSI)
+
+PSI is powered by Lighthouse and includes data from the Chrome User Experience Report (CrUX). It runs on Google Servers, and the high level performance score is === the Lighthouse performance score. The tool allows for in-depth performance analysis of a page during load.
+
+### PSI Performance Score: Lab Data, Blended Score (5 metrics), URL Level
+
+Here we see the top level performance gauge shown in PSI. This score is 0-100 and it is a weighted blend of the lab metrics that are measured by Lighthouse. 
+
+The high level performance score is the same as it is in Lighthouse, calculated from a weighted blend of 6 lab performance metrics. Information about thresholds can be found <a href="https://developers.google.com/web/tools/lighthouse/v3/scoring">here</a>.
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="F4_psi_lab.png" alt="">
```

Comment:
I think these images are missing from the PR. Also you'll want to write descriptive alt text to explain what's going on in the image for non-sighted users.